### PR TITLE
Feature/duplicate avoidance

### PR DESCRIPTION
**Pull request by Frank-Schaeske**

### CODE REVIEW

File: components/List/index.js [hunk 1]
```diff
@@ -3,9 +3,21 @@ import Link from "next/link";
...
+          const nameA = a.player.lastname.toLowerCase();
+          const nameB = b.player.lastname.toLowerCase();
+          if (nameA < nameB) {
+            return -1;
+          }
+          if (nameA > nameB) {
+            return 1;
+          }
+          return 0;
+        })
```
Comment (moonwave99) [inline components/List/index.js:19]: I would outsource the comparator outside the component as:

```js
function comparePlayers(a, b) {
  const nameA = a.player.lastname.toLowerCase();
  const nameB = b.player.lastname.toLowerCase();
  if (nameA < nameB) {
    return -1;
  }
  if (nameA > nameB) {
    return 1;
  }
  return 0;
};
```

Then in JSX just `{players.sort(comparePlayers).map(...)}`

File: components/SearchForm/index.js [hunk 2]
```diff
@@ -45,41 +46,88 @@ export default function SearchForm({ setSearchedPlayer }) {
...
+          <option value="bundesliga2">2. Bundesliga</option>
+        </StyledSelect>
+      </div>
+      <div>
+        <StyledLabel htmlFor="team">Team:</StyledLabel>
+        <StyledSelect name="team" id="team">
+          {germanTeams
+            .filter((germanTeam) => {
+              return germanTeam[season] === league;
+            })
```
Comment (moonwave99) [inline components/SearchForm/index.js:70]: `.filter(germanTeam => germanTeam[season] === league)`

**Comment by Frank-Schaeske** [comment]
Thanks for your feedback, Diego! I will adapt my code accordingly.

File: components/List/index.js [hunk 3]
```diff
@@ -3,42 +3,42 @@ import Link from "next/link";
...
+    const nameB = b.player.lastname.toLowerCase();
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+    return 0;
+  }
+
```
Comment (moonwave99) [inline components/List/index.js]: I would put it outside of the component, so it doesn't get recreated all the time:

```js
function comparePlayers(a, b) { ... }

export default function List({ players }) { ... }
```

Comment (Frank-Schaeske) [reply components/List/index.js]: Thanks again!

**Review by moonwave99 — APPROVED** [review]
LGTM!

**Review by moonwave99 — APPROVED** [review]

**Pull request merged**